.github/workflows/update-oam-defs.yml

```
@@ -0,0 +1,89 @@
+          excludes: prerelease, draft
+      - name: Check out code
+        uses: actions/checkout@master
+        with:
+          fetch-depth: 1
+      - name: Get CURRENT_KUMA_VERSION from .env
+        run: |
+          cat .github/.env > $GITHUB_ENV
+  generator:
+    name: Generate and push OAM definitions
+    needs: [check_kuma_version]
+    if: ${{ needs.check_kuma_version.outputs.NEW_KUMA_VERSION  >  needs.check_kuma_version.outputs.CURRENT_KUMA_VERSION }}
+    env:
+      NEW_KUMA_VERSION: ${{ needs.check_kuma_version.outputs.NEW_KUMA_VERSION }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@master
+        with:
+          fetch-depth: 1
+          token: ${{ secrets.GH_ACCESS_TOKEN }}
+          ref: "master"
+      - name: Get KUMA manifests
+        run: |
+          mkdir ./kuma-crds
+          cd ./kuma-crds
+          curl --remote-name-all `curl -H "Accept: application/vnd.github.VERSION.raw" https://api.github.com/repositories/175425051/contents/deployments/charts/kuma/crds\?ref\=$NEW_KUMA_VERSION| jq 'map(.download_url) | .[]' | tr -d '"'`
+          cd ..
+          awk 'FNR==1 && NR>1 { printf("\n%s\n\n","---") } 1' kuma-crds/*.yaml > ~/kuma.yaml
+      - name: Bulild jsonschema util
+        run: |
+          curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.38.0/install.sh | bash
+          export NVM_DIR="$HOME/.nvm"
+          [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"  # This loads nvm
+          nvm install v14.15.3
+          nvm install-latest-npm
+          git clone https://github.com/layer5io/kubeopenapi-jsonschema util
+          cd util
+          npm i; npm i nexe -g
+          make linux
+      - name: Generate Workload definitions
+        run: |
+          mkdir -p ./templates/oam/workloads-$NEW_KUMA_VERSION
+          cd util
+          template='{"apiVersion":"core.oam.dev/v1alpha1","kind":"WorkloadDefinition","metadata":{"name":""},"spec":{"definitionRef":{"name":""}}}'
+          crds=$(./kubeopenapi-jsonschema --location ~/kuma.yaml -t yaml --filter '$[?(@.kind=="CustomResourceDefinition")]' -o json --o-filter '$..["spec"]["names"]["kind"]' | jq '.[]')
+          for t in ${crds[@]}; do
+            nameUpper=`echo $t  | tr -d '"'`
+            nameLower=`echo $t  | tr -d '"' | tr '[:upper:]' '[:lower:]'`
+            definitionRef=$(printf %s.kuma.meshery.layer5.io $nameLower)
+            name=`echo $t | tr -d '"' | tr '[:upper:]' '[:lower:]'`
+            definitionRef=$(printf %s.kuma.meshery.layer5.io $name)
+            ./kubeopenapi-jsonschema --location ~/kuma.yaml -t yaml --filter '$[?(@.kind=="CustomResourceDefinition" && @.spec.names.kind=='$t')]..openAPIV3Schema.properties.spec' --o-filter "$[]" -o json | jq '.[] | .version = "'$NEW_KUMA_VERSION'" | .title = "'$(printf %sKUMA $nameUpper)'"' > ../templates/oam/workloads-$NEW_KUMA_VERSION/$(printf %s.kuma.meshery.layer5io.schema.json $nameLower)
```

```suggestion
            ./kubeopenapi-jsonschema --location ~/kuma.yaml -t yaml --filter '$[?(@.kind=="CustomResourceDefinition" && @.spec.names.kind=='$t')]..openAPIV3Schema' --o-filter "$[]" -o json | jq '.[] | .version = "'$NEW_KUMA_VERSION'" | .title = "'$(printf %sKUMA $nameUpper)'"' > ../templates/oam/workloads-$NEW_KUMA_VERSION/$(printf %s.kuma.meshery.layer5io.schema.json $nameLower)
```
Try using this. This should fix it.